File: Java/_272.java
/* UVA --> 272 */

import java.util.Scanner;

public class _272 {
    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);
        int i;
        boolean f=true;
        String st1,st2="";
        while(scan.hasNext())
        {
            st1 = scan.nextLine();
            st2 = "";
            for(i=0; i < st1.length(); i++)
            {
                if(st1.charAt(i)==34)
                {
                    if(f==false)
                    {
                        st2+="'";
                        st2+="'";
                        f=true;
                        continue;
                    }
                    st2+="`";
                    st2+="`";
                    f=false;
                    continue;
                }
                st2 += st1.charAt(i);
            }
            System.out.println(st2);
        }
    }
}

File: C/272.c
/*272*/

#include <stdio.h>
#include <string.h>
int main()
{
    int i,j,f=1;
    char st1[999],st2[999];
    while(gets(st1))
    {
        for(i=0,j=0; st1[i]!='\0'; i++,j++)
        {
            if(st1[i]==34)
            {
                if(f==0)
                {
                    st2[j]=39;
                    j++;
                    st2[j]=39;
                    f=1;
                    continue;
                }
                st2[j]=96;
                j++;
                st2[j]=96;
                f=0;
                continue;
            }
            st2[j]=st1[i];
        }
        st2[j]='\0';
        puts(st2);
    }
    return 0;
}
